src/Essentials/src/Screenshot/Screenshot.ios.cs

```diff
@@ -47,25 +34,10 @@ public Task<IScreenshotResult> CaptureAsync(UIWindow window)
 			UIGraphics.BeginImageContextWithOptions(window.Bounds.Size, false, window.Screen.Scale);
 			var ctx = UIGraphics.GetCurrentContext();
 
-			if (!TryRender(window, out var error))
-			{
-				// FIXME: test/handle this case
-			}
-
-			// Render the status bar with the correct frame size
-			try
-			{
-				TryHideStatusClockView(UIApplication.SharedApplication);
-				var statusbarWindow = GetStatusBarWindow(UIApplication.SharedApplication);
-				if (statusbarWindow != null/* && metrics.StatusBar != null*/)
-				{
-					statusbarWindow.Frame = window.Frame;
-					statusbarWindow.Layer.RenderInContext(ctx);
-				}
-			}
```

Comment:
This was the code that actually caused issues, and the try/catch was not enough because Apple just crashes the app if you do bad things.